test/parallel/test-vm-preserves-property.js

```diff
@@ -0,0 +1,23 @@
+var assert = require('assert');
```

Comment:
I had removed it because I was running the test against node 0.10 which didn't recognize all the globals, and then I forgot to put it back.  Restored.